### PR TITLE
Kubernetes Provider: add simple projects for quickstart

### PR DESCRIPTION
**Pull request by linglingye001**
Need update package reference version after `Microsoft.FeatureManagement.AspNetCore` 3.2.0 release.

### CODE REVIEW

File: examples/KubernetesProvider/quickstart-webapp-feature-flag/MyWebApp/Pages/Beta.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.FeatureManagement.Mvc;
+
+namespace MyWebApp.Pages
+{
+    [FeatureGate("Beta")]
```
Comment (RichardChen820) [inline examples/KubernetesProvider/quickstart-webapp-feature-flag/MyWebApp/Pages/Beta.cshtml.cs:6]: Is this only a simple enable/disable scenario? Could we add more scenarios for covering client filters?

**Review by RichardChen820 — APPROVED** [review]

File: examples/KubernetesProvider/quickstart-webapp-feature-flag/MyWebApp/Program.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+using Microsoft.FeatureManagement;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddRazorPages();
+
+// Add a JSON configuration source 
+builder.Configuration.AddJsonFile("config/mysettings.json");
```
Comment (RichardChen820) [inline examples/KubernetesProvider/quickstart-webapp-feature-flag/MyWebApp/Program.cs:9]: reloadOnChange = true

**Review by RichardChen820 — CHANGES_REQUESTED** [review]

File: examples/KubernetesProvider/quickstart-webapp/MyWebApp/Program.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddRazorPages();
+
+// Add a JSON configuration source 
+builder.Configuration.AddJsonFile("config/mysettings.json");
```
Comment (RichardChen820) [inline examples/KubernetesProvider/quickstart-webapp/MyWebApp/Program.cs:7]: reloadOnChange = true

**Comment by linglingye001** [comment]
will put example projects to azure-sample repo

**Pull request closed without merging**